api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -118,4 +118,30 @@
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
     
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi" dbms="mysql">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="additionalDetail" type="VARCHAR(255)">
```

Comment:
Our convention for database table column names is to separate words with hyphen. Just like you did for clinical_status